testsuite/MDAnalysisTests/__init__.py

```diff
@@ -150,28 +147,4 @@
     in_dir,
     assert_nowarns,
 )
-from MDAnalysisTests.core.util import make_Universe
-
-def run(*args, **kwargs):
```

Comment:
it would be nice if you could keep this method. This is shown in our docs how to run the tests yourself from an installation of MDAnalysisTests